src/ComponentDetailsPage.jsx

```diff
@@ -176,166 +381,176 @@ if (!exists) {
-      >
-        Discussion
-      </TabsButton>
-    </Tabs>
-
-    {state.selectedTab === "about" && (
-      <Content>
-        <div>
-          {metadata.description ? (
-            <Markdown text={metadata.description} />
-          ) : (
-            <Text>This component has no description.</Text>
-          )}
-        </div>
-
-        <Sidebar>
-          {(tags.includes("Coming Soon") || tags.includes("coming-soon")) && (
+    <Content>
+      <Wrapper>
+
+        <Tabs>
+          <TabsButton
+            href={`${detailsUrl}&tab=source`}
+            selected={state.selectedTab === "source"}
+          >
+            <Icon className="ph ph-code" />
+            Source & Preview
+          </TabsButton>
+          <TabsButton
+            href={`${detailsUrl}&tab=about`}
+            selected={state.selectedTab === "about"}
+          >
+            <Icon className="bi bi-file-earmark-text" />
+            Read.me
+          </TabsButton>
+          <TabsButton
+            href={`${detailsUrl}&tab=discussion`}
+            selected={state.selectedTab === "discussion"}
+          >
+            <Icon className="bi bi-chat-text" />
+            Discussion
+          </TabsButton>
+        </Tabs>
+
+        {state.selectedTab === "about" && (
+          <Content noSidebar>
             <div>
-              <Widget
-                src="${REPL_ACCOUNT}/widget/WaitList"
-                props={{ formUrl: "http://eepurl.com/hXyUnf" }}
-              />
+              {metadata.description ? (
+                <Markdown text={metadata.description} />
+              ) : (
+                <Text>This component has no description.</Text>
+              )}
             </div>
-          )}
+          </Content>
+        )}
 
-          <div>
-            <SmallTitle>Developer</SmallTitle>
+        {state.selectedTab === "source" && (
+          <Content noSidebar>
             <Widget
-              src="${REPL_ACCOUNT}/widget/AccountProfile"
+              src="${REPL_ACCOUNT}/widget/ComponentHistory"
+              props={{ widgetPath: src }}
+            />
+          </Content>
+        )}
+
+        {state.selectedTab === "discussion" && (
+          <Content noSidebar>
+            <Widget
+              src="${REPL_ACCOUNT}/widget/NestedDiscussions"
               props={{
-                accountId: accountId,
+                identifier: src,
+                notifyAccountId: accountId,
+                parentComponent: "${REPL_ACCOUNT}/widget/ComponentDetailsPage",
+                parentParams: { tab: "discussion", src },
+                highlightComment: props.highlightComment,
               }}
             />
-          </div>
-
-          {tags.length > 0 && (
-            <div>
-              <SmallTitle>Tags</SmallTitle>
+          </Content>
+        )}
+      </Wrapper>
+      <Sidebar>
+        <SideBarContainer>
+          <SmallTitle style={{ "padding-top": "20px" }}>DEVELOPER</SmallTitle>
```

Comment:
Any time we want to render a string in all uppercase, we should rely on CSS to do the transformation with `text-transform: uppercase` and write the text as `Developer`. Writing the text in all uppercase can cause accessibility issues for screen readers.